cypress/locators/preview/locators.js

```diff
@@ -0,0 +1,3 @@
+// component preview locators
+export const PREVIEW_COMPONENT = '[data-component="preview"]';
+export const ROOT_COMPONENT = "#__cy_root";
```

Comment:
this is already used in `action-popover.test.js` file. So this is not good place to store it here. I would go with `common` locators and just use it in both?